sdks/python/test-suites/containers/tensorrt_runinference/tensor_rt.dockerfile

```diff
@@ -22,8 +22,8 @@ ENV PATH="/usr/src/tensorrt/bin:${PATH}"
 
 WORKDIR /workspace
 
-RUN pip install --no-cache-dir apache-beam[gcp]==2.40.0
-COPY --from=apache/beam_python3.8_sdk:2.40.0 /opt/apache/beam /opt/apache/beam
+RUN pip install --no-cache-dir apache-beam[gcp]==2.42.0
```

Comment:
Just double checking.  Does having this versioned mean we'll be running the test against only the same version?  wouldn't we want this to run against head?

Reply:
The version is needed for the multi stage process. We need to copy necessary artifacts from the base image to the custom container image. 

https://beam.apache.org/documentation/runtime/environments/#modify-existing-base-image

Also, as part of the benchmark workflow, we make the sdist of the `head` and install it on the worker. Any changes made on the master will be reflected when we install from the `apache_beam-2.xx.x.dev0.tar` on the workers.

![image](https://user-images.githubusercontent.com/34158215/204897990-c1993bd1-57cc-4270-a051-5cf515c304a3.png)